syfertext/doc.py

```diff
@@ -155,22 +164,26 @@ def __getitem__(self, key: Union[int, slice]) -> Union[Token, Span, int]:
 
             return span
 
-    def __len__(self):
+    @type_hints
+    def __len__(self) -> int:
         """Return the number of tokens in the Doc."""
         return len(self.container)
 
-    def __iter__(self):
+    @type_hints
+    def __iter__(self) -> Iterator[Token]:
         """Allows to loop over tokens in `self.container`"""
         for i in range(len(self.container)):
 
             # Yield a Token object
             yield self[i]
 
     @property
-    def text(self):
+    @type_hints
+    def text(self) -> str:
         """Returns the text present in the doc with whitespaces"""
         return "".join(token.text_with_ws for token in self)
 
+    # Find Return Type: Vector
```

Comment:
when master is merged into this branch it would be `torch.Tensor` but currently its np.array so let's keep this on hold